js/ui/status/missionGame.js

```diff
@@ -0,0 +1,648 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+
+const Clutter = imports.gi.Clutter;
+const GLib = imports.gi.GLib;
+const Lang = imports.lang;
+const St = imports.gi.St;
+const Signals = imports.signals;
+
+const BoxPointer = imports.ui.boxpointer;
+const PanelMenu = imports.ui.panelMenu;
+const PopupMenu = imports.ui.popupMenu;
+
+const MissionGameService = imports.misc.missionGameService;
+
+function launchLessonAction(lesson) {
+    return function() {
+        /* XXX: This needs to spawn a wrapper script that goes through each lesson
+         * individually as opposed to just running showmehow. */
```

Comment:
I feel that this should be somehow coordinated with the service over DBus, or something a bit more robust than this.